### ISSUE TITLE
Add ability to mute fanfare

### ISSUE THREAD

**Issue by grrrrr** [billthefarmer/gurgle#29]
with the release of 1.08 a fan fare was added.  It would be appreciated to have the option to mute/disable this as it is quite intrusive

**Comment by billthefarmer** [comment]
It's possible to please some of the the people most of the time, but not possible to please all of the people all the time. See #18.

Use Settings->Sound and turn media volume down. You can always adjust it later with the volume up/down buttons while listening to music.

**Comment by grrrrr** [comment]
> not possible to please all of the people all the time

It literally is in this case, see my requested feature above ;)

> Use Settings->Sound and turn media volume down. You can always adjust it later with the volume up/down buttons while listening to music

Not being expected to use other media while playing gurgle is a bit of a stretch, no?

**Comment by c8c8** [comment]
Agreed.  If you're going to have sounds, you need to have mute.  While I enjoy the fanfare, when I'm at work playing this on the toilet, people think I'm celebrating just a bit too much in the stall.

**Comment by billthefarmer** [comment]
OK, so if you are ensconced in the khazi, listening to music, you will be able to disable the fanfare. And you won't get a round of applause when you come out. That will be in the next version.
